conda/conda-build/meta.yaml

```diff
@@ -98,6 +98,8 @@ requirements:
     - cmake {{ cmake_version }}
     - {{ compiler('c') }} =11.2
     - {{ compiler('cxx') }} =11.2
+    - elfutils # [linux]
```

Comment:
CMake is not able to find libdw during the conda package build process.

Reply:
Can you confirm that packages `libdwarf` and `elfutils` are included in the conda enviroment, and it is activated before the build starts?